plugins/chat/assets/javascripts/discourse/components/chat-message.js

```diff
@@ -86,6 +96,25 @@ export default class ChatMessage extends Component {
   @action
   teardownChatMessage() {
     cancel(this._invitationSentTimer);
+
+    this.args.message.mentionedUsers.forEach((user) => {
```

Comment:
Can we abstract all of this in a method of message? teardownMentionedUsers ?

Reply:
That will be good, done.